9/CHANGELOG-v9.rst

```diff
@@ -209,6 +1056,12 @@ community.zabbix
 - zabbix_api_info module added
 - zabbix_user module - add current_passwd optional parameter to enable password updating of the currently logged in user (https://www.zabbix.com/documentation/6.4/en/manual/api/reference/user/update)
 
+dellemc.openmanage
+~~~~~~~~~~~~~~~~~~
+
+- Ansible lint issues are fixed for the collections.
+- Module ``redfish_storage_volume`` is enhanced to support reboot options and job tracking operation.
+
```

Comment:
here

Reply:
This usually happens if collections have a new release in which they also modify changelog entries for older releases. For example, for dellemc.openmanage, which was updated from 8.5.0 to 8.7.0, look at the comparison: https://github.com/dell/dellemc-openmanage-ansible-modules/compare/v8.5.0..v8.7.0#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43c - you can see that they modified the 8.5.0 changelog as well *after* the 8.5.0 release. If you look at the diff for changelogs/changelog.yaml, you can see that they fixed a broken section name, and a typo in a corresponding fragment.